tests/integration/security/external_ca_istiodcerts/main_test.go

```diff
@@ -0,0 +1,110 @@
+// +build integ
+//  Copyright Istio Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package externalca
+
+import (
+	"testing"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/echo"
+	"istio.io/istio/pkg/test/framework/components/echo/echoboot"
+	"istio.io/istio/pkg/test/framework/components/istio"
+	"istio.io/istio/pkg/test/framework/components/namespace"
+	"istio.io/istio/pkg/test/framework/label"
+	"istio.io/istio/pkg/test/framework/resource"
+	"istio.io/istio/tests/integration/security/util"
+)
+
+const (
+	ASvc = "a"
+	BSvc = "b"
+)
+
+type EchoDeployments struct {
+	Namespace namespace.Instance
+	// workloads for TestSecureNaming
+	A, B echo.Instances
+}
+
+var (
+	inst istio.Instance
+	apps = &EchoDeployments{}
+)
+
+func SetupApps(ctx resource.Context, apps *EchoDeployments) error {
+	var err error
+	apps.Namespace, err = namespace.New(ctx, namespace.Config{
+		Prefix: "test-ns",
+		Inject: true,
+	})
+	if err != nil {
+		return err
+	}
+
+	builder := echoboot.NewBuilder(ctx)
+	builder.
+		WithClusters(ctx.Clusters()...).
+		WithConfig(util.EchoConfig(ASvc, apps.Namespace, false, nil)).
+		WithConfig(util.EchoConfig(BSvc, apps.Namespace, false, nil))
+
+	echos, err := builder.Build()
+	if err != nil {
+		return err
+	}
+	apps.A = echos.Match(echo.Service(ASvc))
+	apps.B = echos.Match(echo.Service(BSvc))
+	return nil
+}
+
+func TestMain(m *testing.M) {
+	// Integration test for testing interoperability with external CA's that are integrated with K8s CSR API
+	// and sign istiod certs as well
+	// Refer to https://kubernetes.io/docs/reference/access-authn-authz/certificate-signing-requests/
+	framework.NewSuite(m).
+		Label(label.CustomSetup).
+		RequireMinVersion(18).
+		// https://github.com/istio/istio/issues/22161. 1.22 drops support for legacy-unknown signer
+		RequireMaxVersion(21).
```

Comment:
Seems not great to add a new test that we already know is broken on Kubernetes latest version..

Reply:
In the future after we figure out how to support different k8s signers we can enable these cases back.

Reply:
Not sure what the purpose of both the tests are.

 There is already a test that uses chiron (legacy-unknown signer) to sign DNS certs in tests/integration/security/ chiron. 

There is also a test that does e2e reachability check in tests/integration/security/ chiron/external_ca that does mTLS reachability checks when the workload certs are signed by K8s default signer

Reply:
Yes. The case you mentioned is to cover 1 scenario for each: one is to cover DNS cert and the other is to cover workload certs, So both of them have CA server enabled.   This new test  is to cover the `AND` scenario:  both istiod DNS cert and workload certs are signed from legacy-unknown signer and CA server is disabled.